cms/apps/pages/templatetags/pages.py

```diff
@@ -7,22 +7,15 @@
 
 from cms.apps.pages.models import Page
 from cms.models import SearchMetaBase
+from django_jinja import library
+
+import jinja2
 
 register = template.Library()
 
 
 # Navigation.
-
-@register.inclusion_tag("pages/navigation.html", takes_context=True)
-def navigation(context, pages, section=None):
-    """
-    Renders a navigation list for the given pages.
-
-    The pages should all be a subclass of PageBase, and possess a get_absolute_url() method.
-
-    You can also specify an alias for the navigation, at which point it will be set in the
-    context rather than rendered.
-    """
+def navigation_entries(context, pages, section=None, is_json=False):
```

Comment:
What about prefixing with just an underscore to mark it as an internal function?